datev_export_xml/tests/test_datev_export.py

```diff
@@ -0,0 +1,868 @@
+                ("type", "=", "purchase"),
+                ("company_id", "=", cls.company.id),
+            ],
+            limit=1,
+        )
+        if not cls.purchase_journal:
+            cls.purchase_journal = cls.JournalObj.create(
+                {
+                    "name": "Test purchase journal",
+                    "code": "purchase",
+                    "type": "purchase",
+                    "company_id": cls.company.id,
+                }
+            )
+        cls.AccountObj = cls.env["account.account"]
+        cls.PartnerObj = cls.env["res.partner"]
+        cls.AnalyticAccountObj = cls.env["account.analytic.account"]
+        cls.ProductObj = cls.env["product.product"]
+        cls.today = fields.Date.today()
+        cls.InvoiceObj = cls.env["account.move"]
+        cls.InvoiceLineObj = cls.env["account.move.line"]
+        cls.AttachmentObj = cls.env["ir.attachment"]
+        cls.DatevExportObj = cls.env["datev.export.xml"]
+
+        cls.inv_attach_de = cls.env.ref("datev_export_xml.vendor_bill_attachment_DE")
+        cls.inv_attach_eu = cls.env.ref("datev_export_xml.vendor_bill_attachment_EU")
+        cls.inv_attach_noneu = cls.env.ref(
+            "datev_export_xml.vendor_bill_attachment_NonEU"
+        )
+
+        cls.refund_attach_de = cls.env.ref("datev_export_xml.bill_refund_attachment_DE")
+        cls.refund_attach_eu = cls.env.ref("datev_export_xml.bill_refund_attachment_EU")
+        cls.refund_attach_noneu = cls.env.ref(
+            "datev_export_xml.bill_refund_attachment_NonEU"
+        )
+
+        cls.customer_de = cls.env.ref("datev_export_xml.customer_DE")
+        cls.vendor_de = cls.env.ref("datev_export_xml.vendor_DE")
+
+        cls.customer_eu = cls.env.ref("datev_export_xml.customer_EU")
+        cls.vendor_eu = cls.env.ref("datev_export_xml.vendor_EU")
+
+        cls.customer_noneu = cls.env.ref("datev_export_xml.customer_NonEU")
+        cls.vendor_noneu = cls.env.ref("datev_export_xml.vendor_NonEU")
+
+        cls.account_income = cls.env.ref("datev_export_xml.account_datev_income")
+        cls.account_expense = cls.env.ref("datev_export_xml.account_datev_expense")
+
+        cls.consulting = cls.env.ref("datev_export_xml.product_datev_01")
+        cls.lease = cls.env.ref("datev_export_xml.product_datev_02")
+
+        cls.analytic_account_it = cls.env.ref(
+            "datev_export_xml.analytic_account_datev_01"
+        )
+        cls.analytic_account_office = cls.env.ref(
+            "datev_export_xml.analytic_account_datev_02"
+        )
+
+        cls.parent_customer = cls.env.ref("datev_export_xml.customer_parent")
+        cls.child_customer = cls.env.ref("datev_export_xml.customer_child")
+
+        # Ensure the initial state
+        cls.refund_date = cls.today - timedelta(days=55)
+        cls.start_date = cls.today - timedelta(days=34)
+        cls.end_date = cls.today - timedelta(days=32)
+        cls.InvoiceObj.with_context(force_delete=True).search([]).unlink()
```

Comment:
This line is problematic if we have other modules installed. 

For example, if you install sale and try to pass the test, it will fail (no other modules installed). The problem comes from the fact that odoo auto installs account_audit_trail by default on `l10n_de`.

The way to solve it is to add 
```

Can you add it?

        if "check_account_audit_trail" in cls.env["res.company"]._fields:
            cls.env["res.company"].search([]).write({"check_account_audit_trail": False})
```